wpilibNewCommands/src/main/native/cpp/frc2/command/WaitCommand.cpp

```diff
@@ -22,7 +30,7 @@ void WaitCommand::End(bool interrupted) {
 }
 
 bool WaitCommand::IsFinished() {
-  return m_timer.HasElapsed(m_duration);
+  return m_timer.HasElapsed(*m_delaySeconds.target<units::second_t>());
```

Comment:
```suggestion
  return m_timer.HasElapsed(m_delaySeconds());
```
Why were you using [`target`](https://en.cppreference.com/w/cpp/utility/functional/function/target)? The docs indicate that it is used for checking if the underlying *function* type matches the template argument, not the return type.
This is also what's causing the errors, since the docs indicate that it will return a null pointer if the type doesn't match, which is always the case.